SECURITY.md

```diff
@@ -2,11 +2,11 @@
 
 ## Supported Versions
 
-Please vist the [Releases](https://struts.apache.org/releases.html#prior-releases) page to see full information about each version 
+Please visit the [Releases](https://struts.apache.org/releases.html#prior-releases) page to see full information about each version 
 and what potential vulnerability it can have:
 
 | Version | Supported          |
-| ------- | ------------------ |
+|---------|--------------------|
 | 6.0.0   | :white_check_mark: |
```

Comment:
IMHO it would be more consistent to mention just the maintenance branches (e.g., 6.x and 2.5.x) instead of specific versions that need permanent updates to this table?

@lukaszlenart I could add another commit if you find this valuable.

Reply:
Right, make sense. We have the release page with detailed info as well.

Reply:
WIP

Reply:
done